js/main.js

```diff
@@ -1,21 +1,86 @@
-const getRandomIntInclusive = function (min, max) {
-  // Функция взята с https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
-  if (min >= 0 && max >= 0 && min !== max && min < max) {
-    min = Math.ceil(min);
-    max = Math.floor(max);
-    return Math.floor(Math.random() * (max - min + 1)) + min; //Максимум и минимум включаются
-  }
-  return('Введите верные значения диапазона');
+function getRandomPositiveFloat (a, b, digits = 1) {
+  const lower = Math.min(Math.abs(a), Math.abs(b));
+  const upper = Math.max(Math.abs(a), Math.abs(b));
+  const result = Math.random() * (upper - lower) + lower;
+  return result.toFixed(digits);
+}
+
+function getRandomPositiveInteger (a, b) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+}
+
+const TYPES_OF_HOUSING = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const CHECKIN_TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const CHECKOUT_TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const ALL_FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+]
+
+const ALL_PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const SIMILAR_COUNT = 10;
+
+const getRandomArrayElement = (elements) => {
+  return elements[getRandomPositiveInteger(0, elements.length - 1)];
```

Comment:
```suggestion
  const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
```

стрелочная функция по умолчанию делает ретурн. Блок кода нужен тогда, когда больше одной операции нужно проделать.